internal/services/recoveryservices/backup_policy_file_share_resource_test.go

```diff
@@ -467,6 +508,66 @@ resource "azurerm_backup_policy_file_share" "test" {
 `, r.template(data), data.RandomInteger)
 }
 
+func (r BackupProtectionPolicyFileShareResource) vaultStandard(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_backup_policy_file_share" "test" {
+  name                = "acctest-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  recovery_vault_name = azurerm_recovery_services_vault.test.name
+
+  backup {
+    frequency = "Hourly"
+
+    hourly {
+      interval        = 4
+      start_time      = "10:00"
+      window_duration = 12
+    }
+  }
+
+  retention_daily {
+    count = 10
+  }
+
+  snapshot_retention_in_days = 9
+
+  backup_tier = "vault-standard"
+}
+`, r.template(data), data.RandomInteger)
+}
+
+func (r BackupProtectionPolicyFileShareResource) vaultStandardUpdated(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_backup_policy_file_share" "test" {
+  name                = "acctest-%d"
+  resource_group_name = azurerm_resource_group.test.name
+  recovery_vault_name = azurerm_recovery_services_vault.test.name
+
+  backup {
+    frequency = "Hourly"
+
+    hourly {
+      interval        = 4
+      start_time      = "10:00"
+      window_duration = 12
+    }
+  }
+
+  retention_daily {
+    count = 20
+  }
+
+  snapshot_retention_in_days = 15
+
+  backup_tier = "vault-standard"
```

Comment:
is it possible to switch between `vault-standard` and `snapshot`, and if so, can we test this?

Reply:
Sure. Will add one more step.